plenum/common/request.py

```diff
@@ -43,12 +43,15 @@ def digest(self):
     @property
     def as_dict(self):
         rv = {
-            f.IDENTIFIER.nm: self._identifier,
             f.REQ_ID.nm: self.reqId,
             OPERATION: self.operation,
-            f.SIG.nm: self.signature,
-            f.SIGS.nm: self.signatures
         }
+        if self._identifier is not None:
```

Comment:
Do we need this checks?

Reply:
these checks were added to correctly cooperate with sdk signing function - it ignores actual value of field and just checks the presence